.github/actions/javascript/contributorChecklist/contributorChecklist.js

```diff
@@ -137,16 +146,19 @@ getPullRequestBody()
         let authorChecklistComplete = false;
         let reviewerChecklistComplete = false;
 
-        // Once we've gathered all the data, loop through each comment and look to see if it contains a completed checklist
+        const authorChecklistNumber = countCheckedBoxes(completedAuthorChecklist);
+        const reviewerChecklistNumber = countCheckedBoxes(completedReviewerChecklist);
+
+        // Once we've gathered all the data, loop through each comment and look to see if it contains enough completed checkboxes
         for (let i = 0; i < combinedData.length; i++) {
-            const whitespace = /([\n\r])/gm;
-            const comment = combinedData[i].replace(whitespace, '');
+            const comment = combinedData[i];
+            const commentChecklistNumber = countCheckedBoxes(comment);
 
-            if (comment.includes(completedAuthorChecklist.replace(whitespace, ''))) {
+            if (commentChecklistNumber >= authorChecklistNumber - 2 && commentChecklistNumber <= authorChecklistNumber + 2 && !comment.includes('[ ]')) {
                 authorChecklistComplete = true;
             }
 
-            if (comment.includes(completedReviewerChecklist.replace(whitespace, ''))) {
+            if (commentChecklistNumber >= reviewerChecklistNumber - 2 && commentChecklistNumber <= reviewerChecklistNumber + 2 && !comment.includes('[ ]')) {
```

Comment:
With this logic if the author and reviewer checklists will have around the same number of checkboxes, just one of them filled in would make the others pass too.

We could maybe save the iterator `i` of the one which passed and then make sure the temp_i != i in the second condition or something like this

Reply:
That could work! Or if you think it makes sense above to just also check for the words `PR Author Checklist` or `PR Reviewer Checklist` we could do that too. I'm pretty sure people always copy the header along with the list, or we could at worst specify that they have to

Reply:
I think we can avoid the specific prefixes at least imo

Reply:
Followed your suggestion here so I think we should be all set!